src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -21,6 +21,15 @@
  */
 abstract class WebTestCase extends KernelTestCase
 {
+    use WebTestAssertions;
+
+    protected function tearDown()
+    {
```

Comment:
you need to also call the parent method, otherwise the kernel is not shutdown properly

Reply:
Fixed in 86fdbb8